gp-includes/formats/format-android.php

```diff
@@ -10,6 +10,13 @@ class GP_Format_Android extends GP_Format {
 	public function print_exported_file( $project, $locale, $translation_set, $entries ) {
 		$this->exported = '';
 		$this->line( '<?xml version="1.0" encoding="utf-8"?>' );
+
+		$this->line( '<!--' );
+		$this->line( 'Translation-Revision-Date: ' . GP::$translation->last_modified( $translation_set ) . '+0000' );
+		$this->line( "Plural-Forms: nplurals={$locale->nplurals}; plural={$locale->plural_expression};" );
+		$this->line( 'X-Generator: GlotPress/' . GP_VERSION );
```

Comment:
See https://github.com/swissspidy/glotpress-json/issues/4#issuecomment-252355310 for the `X-` prefix.

Shouldn't we add the language too?